packages/tokens/properties/size/font.json

```diff
@@ -104,6 +104,10 @@
           "value": "2rem",
           "comment": "Line height for Heading 4 sized text"
         },
+        "h5": {
```

Comment:
oh, I excluded this intentionally because it was the exact same as the `body` value. was thinking that "tokens" should have unique values, otherwise it's a bit confusing which one to use

Reply:
Yeah, I'm being a little inconsistent here, but I think having them line up, even if they're the same means we can
- Distinguish them more easily in config changes
- More consistently reference them in our tailwind config for `text-h5`

Since this is still an alpha change, I can revert if the discussion ends up changing.